src/vm/arm64/cgencpu.h

```diff
@@ -191,9 +191,10 @@ inline BOOL ClrFlushInstructionCache(LPCVOID pCodeAddr, size_t sizeOfCode)
 }
 
 //------------------------------------------------------------------------
-inline void emitJump(UINT32* pCode, LPVOID target)
+inline void emitJump(LPBYTE pBuffer, LPVOID target)
 {
     LIMITED_METHOD_CONTRACT;
+    UINT32* pCode = (UINT32*)pBuffer;
```

Comment:
Unless I am missing something, this cast does not look correct. emitJump was using UINT32* since it was being called only for 32bit platforms in jitinterface.cpp (see your change below). Now that you intend to use it for Arm64, you will need to account for it.